pdl-compiler-fuzz/Cargo.toml

```diff
@@ -11,14 +11,7 @@ cargo-fuzz = true
 libfuzzer-sys = "0.4"
 
 [dependencies.pdl-compiler]
-path = ".."
-
-# Prevent this from interfering with workspaces
-[workspace]
-members = ["."]
-
-[profile.release]
-debug = 1
```

Comment:
You seem to have lost the `debug = 1` line?

Reply:
It is moved to the main Cargo.toml as it is not allowed to have profile information in package definitions.